packages/icons/src/index.js

```diff
@@ -119,5 +119,6 @@ export { default as update } from './library/update';
 export { default as upload } from './library/upload';
 export { default as verse } from './library/verse';
 export { default as video } from './library/video';
+export { default as videoAlt2 } from './library/video-alt2';
```

Comment:
That's probably not the best name for the icon as we're trying to be as descriptive as possible with the icons package. cc @jasmussen here

Reply:
Yep. We need a new name and a new icon. I'll provide both when I have a chance.

Reply:
Have changed the name with 654924b. Will mark this feedback as "Resolved" but please, feel free to comment more if needed, thanks!